relay-server/src/services/processor/span/processing.rs

```diff
@@ -293,9 +288,12 @@ fn normalize(
         ProcessingState::root(),
     )?;
 
+    if let Some(span) = annotated_span.value() {
+        validate_span(span)?;
+    }
     process_value(
         annotated_span,
-        &mut TransactionsProcessor::new(Default::default(), Some(timestamp_range)),
```

Comment:
Calling the processor with a span results in only running `process_span`, so `timestamp_range` checks were noops before. The remaining timestamp checks are done in the `validate_span` call above.
cc @jjbayer

Reply:
Thanks for this!